benchmarks/sm90_push_megamoe_baseline.py

```
@@ -0,0 +1,185 @@
+    return _runner
+
+
+def quant_weights(w13: torch.Tensor, w2: torch.Tensor):
+    """Per-expert 128x128 block quant, cached by weight identity (offline cost)."""
+    key = w13.data_ptr()
+    if key in _weight_cache:
+        return _weight_cache[key]
+    from flashinfer.testing.utils import per_block_cast_to_fp8
+
+    E, two_i, H = w13.shape
+    _, H2, I = w2.shape
```

Ruff (0.16.0)

[error] Ambiguous variable name: `I`

(E741)